pkg/apis/seedmanagement/helper/helper.go

```diff
@@ -76,6 +98,15 @@ func getDecoder(withDefaulter bool) runtime.Decoder {
 
 }
 
+func getEncoder(gv runtime.GroupVersioner, mediaType string) (runtime.Encoder, error) {
```

Comment:
Nit: both paramters are unused.

Reply:
Thanks, overlooked this, should be fixed now.